view-user/src/components/common/Loading/Loading.module.css

```
@@ -0,0 +1,58 @@
+/* Loading */
+.container {
+  background-color: white;
+  margin: 9.375vw;
+  border-radius: 10vw;
+}
+
+.main {
+  display: flex;
+  flex-flow: column;
```

```suggestion
  flex-direction: column;
```
`flex-flow: column wrap`とかでflex-direction と flex-wrap の両方を一度に設定することができるらしい。初めて知った。けど今はcolumnしかつけてなくてflex-directionと同義なので修正してください。